repository/repository.go

```diff
@@ -0,0 +1,9 @@
+package repository
+
+import "database/sql"
+
+type repository struct {
```

Comment:
Related to https://github.com/sungindra/go-dummy-proj/pull/6/files#r1479869000, I think since this will be passed down to handlers, better make it public?
```suggestion
type Repository struct {
```